CHANGELOG.rst

```diff
@@ -7,16 +7,18 @@ x.x.x (TBD)
 
 * Added Discrete panel
 * Added support for colors in stat mapping panel with StatValueMappings & StatRangeMappings
+* Added missing auto interval properties in Template
 
 Changes
 -------
 
 * Refine expectations of is_color_code
 * Deprecated StatMapping, StatValueMapping & StatRangeMapping
+* Change YAxis min value default from None to 0
```

Comment:
It was chnaged from `0 -> None`: https://github.com/weaveworks/grafanalib/pull/399/files#diff-e183577edc3e65361ecd131b1d852e4b241b30f61edd379a745317aa49502339R490 not the other way round.

Reply:
Good spot, will change this back in one of the other PR's I have open

Reply:
There you go: https://github.com/weaveworks/grafanalib/pull/407/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cR17 ;)